_data/lessons.yml

```diff
@@ -0,0 +1,31 @@
+---
+- id      : intro
+  title   : Introduction to High-Performance Computing
+  repo    : hpc-intro
+  site_url: https://carpentries-incubator.github.io
+  repo_url: https://github.com/carpentries-incubator
+  maintain: The HPC Carpentry Community
+- id      : shell
+  title   : Introduction to Using the Shell in a High-Performance Computing Context
+  repo    : hpc-shell
+  site_url: http://hpc-carpentry.org
+  repo_url: https://github.com/hpc-carpentry
+  maintain: The HPC Carpentry Community
+- id      : parallel
+  title   : Introduction to Parallelisation on HPC Platforms
+  repo    : hpc-parallel-novice
+  site_url: http://hpc-carpentry.org
+  repo_url: https://github.com/hpc-carpentry
+  maintain: The HPC Carpentry Community
+- id      : python
+  title   : Introduction to High-Performance Computing in Python
```

Comment:
We should make sure to match these up with the actual lesson titles, I synched this via https://github.com/hpc-carpentry/hpc-python/commit/8dfdd56cd706650a9089b6f11e9a6ff5b5d02e22 (which I accidentally pushed directly rather than open a PR)